ssh/sshd_config

```diff
@@ -29,9 +29,9 @@ StrictModes yes
 
 RSAAuthentication yes
 PubkeyAuthentication yes
-#AuthorizedKeysFile	%h/.ssh/authorized_keys
-AuthorizedKeysCommand /usr/bin/sss_ssh_authorizedkeys
-#AuthorizedKeysCommand /usr/bin/provisor_authorized_keys
+
+AuthorizedKeysFile        none
```

Comment:
it's apparently a thing: https://lists.mindrot.org/pipermail/openssh-unix-dev/2015-July/034186.html